examples/badDisk.pp

```diff
@@ -0,0 +1,11 @@
+$zonea = 'us-central1-a'
+
+gce_auth { 'upbeat-airway-600':
+}
+
+gce_disk { 'mary':
```

Comment:
Doesn't need quotes around variable, and arrows not aligned. 

It's probably worth linting all the examples. I'll hold off commenting on all of them individually for the moment.